src/Content/PlainTextContentStorageHandler.php

```diff
@@ -120,24 +119,20 @@ public function modified(string $version, string $lang): int|null
 	 *
 	 * @param string $lang Code `'default'` in a single-lang installation
 	 * @return array<string, string>
-	 *
-	 * @throws \Kirby\Exception\NotFoundException If the version does not exist
 	 */
-	public function read(string $version, string $lang): array
+	public function read(VersionId $versionId, string $lang): array
 	{
-		return Data::read($this->contentFile($version, $lang));
+		return Data::read($this->contentFile($versionId, $lang));
 	}
 
 	/**
 	 * Updates the modification timestamp of an existing version
 	 *
 	 * @param string $lang Code `'default'` in a single-lang installation
-	 *
-	 * @throws \Kirby\Exception\NotFoundException If the version does not exist
```

Comment:
The old exception type was wrong, but shouldn't we still keep documenting the Exception thrown in this method?

Reply:
What about the exception on error?